src/bosh_azure_cpi/spec/unit/helpers_spec.rb

```diff
@@ -150,10 +159,10 @@ def set_logger(logger)
   end
 
   describe '#get_arm_endpoint' do
-    let(:azure_config) { instance_double(Bosh::AzureCloud::AzureConfig) }
+    let(:azure_config) { instance_double(Bosh::AzureCloud::AzureConfig(vm_config)) }
```

Comment:
This syntax is confusing, is `AzureConfig` a method, or is this a special `instance_double` behavior?

Reply:
Yeah, that's totally not valid... Reason it wasn't causing failures is the let() is overridden in all contexts. Deleting this line entirely.